### ISSUE TITLE
Top level bunches appearing in 'Bunches' submenu

### ISSUE THREAD

**Issue by stevelw** [ttscoff/bunch#193]
After latest update all my top level bunches appear in a 'Bunches' submenu. 

As per documentation and previous behaviour, top level bunches should appear on the main Bunches menu, with numeric keyboard shortcuts. 

V1.4.7

macOS 12.2

**Comment by ttscoff** [comment]
Is there any chance you have your Bunch folder set to a folder which contains the folder “Bunches”?

Thanks,
Brett

**Comment by stevelw** [comment]
No, I already checked that and hadn't changed the setting anytime recently. 

I think this was caused by a OneDrive update (which is where my folder is), which caused access controls to be reset across the system. I had already reauthorised access for Bunch. 

Looking at this now I found I could change the folder preference using the drop down options BUT then reselecting my normal folder wouldn't take when clicked on. 

So I added the folder again, and now everything's back to normal. 

So, it seems it was a weird OneDrive folder permission issue where everything in Bunch worked normally except the menu behaviour described.
